hrjobcontract/CRM/Hrjobcontract/DAO/HRJobLeave.php

```diff
@@ -197,6 +211,7 @@ static function &fieldKeys()
                 'id' => 'id',
                 'leave_type' => 'hrjobcontract_leave_leave_type',
                 'leave_amount' => 'hrjobcontract_leave_leave_amount',
+                'add_public_holidays' => 'hrjobcontract_leave_add_public_holidays',
```

Comment:
I don't think so. The prefix is just **hrjobcontract_leave** and the field name if **add_public_holidays**, hence hrjobcontract_leave_add_public_holidays. For a coincidence the other field names start with **leave** and that is why we have it twice.